lib/public/components/common/navigation/breadcrumbs.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h, iconChevronRight } from '/js/src/index.js';
+
+/**
+ * Create Breadcrumbs out of provided items, each items is preceded by rightChevron Icon
+ * @param {Component[]} items items in breadcrumbs
+ * @return {Component} breadcrumbs
+ */
+export const breadcrumbs = (...items) => h('.flex-row.g1.items-center', items.flatMap((item) => [iconChevronRight(), item]));
```

Comment:
Why `...` items and not simply take an array?
I think that you should not put chevron in the first component, the breadcrumbs should include all the items

Reply:
1. refactored breadcrumbs so it doesn't put chevron as first element.
2. I would keep parameters with spread operator, as there is no particular reason to use array, it would required one additional code element - array constructor, it would be more handy if breadcrumbs were supposed to work with many elements, naturally being held in array, but it's not

Reply:
refactored not to use spread operator